src/test/java/uk/gov/companieshouse/certifiedcopies/orders/api/controller/CertifiedCopiedControllerIntegrationTest.java

```diff
@@ -0,0 +1,157 @@
+                .andExpect(jsonPath("$.item_options.contact_number", is(CONTACT_NUMBER)))
+                .andExpect(jsonPath("$.item_options.delivery_method", is(DeliveryMethod.POSTAL.getJsonName())))
+                .andExpect(jsonPath("$.item_options.delivery_timescale", is(DeliveryTimescale.STANDARD.getJsonName())))
+                .andExpect(jsonPath("$.item_options.filing_history_documents[0].filing_history_id", is(FILING_HISTORY_ID)))
+                .andExpect(jsonPath("$.item_options.forename", is(FORENAME)))
+                .andExpect(jsonPath("$.item_options.surname", is(SURNAME)))
+                .andExpect(jsonPath("$.kind", is(KIND)))
+                .andExpect(jsonPath("$.postal_delivery", is(true)))
+                .andExpect(jsonPath("$.quantity", is(QUANTITY)));
+    }
+
+    @Test
+    @DisplayName("Successfully creates certified copy item with default delivery method and delivery timescale")
+    void createCertifiedCopyDefaultsDeliveryMethodAndDeliveryTimeScale() throws Exception {
+        final CertifiedCopyItemRequestDTO certifiedCopyItemDTORequest = new CertifiedCopyItemRequestDTO();
+        certifiedCopyItemDTORequest.setCompanyNumber(COMPANY_NUMBER);
+        certifiedCopyItemDTORequest.setQuantity(QUANTITY);
+
+        final CertifiedCopyItemOptionsRequestDTO certifiedCopyItemOptionsDTORequest = new CertifiedCopyItemOptionsRequestDTO();
+        certifiedCopyItemDTORequest.setItemOptions(certifiedCopyItemOptionsDTORequest);
+
+        mockMvc.perform(post(CERTIFIED_COPIED_URL)
+                .header(REQUEST_ID_HEADER_NAME, REQUEST_ID_VALUE)
+                .header(ERIC_IDENTITY_TYPE, ERIC_IDENTITY_TYPE_OAUTH2_VALUE)
+                .header(ERIC_IDENTITY, ERIC_IDENTITY_VALUE)
+                .contentType(MediaType.APPLICATION_JSON)
+                .content(objectMapper.writeValueAsString(certifiedCopyItemDTORequest)))
+                .andExpect(status().isCreated())
+                .andExpect(jsonPath("$.company_number", is(COMPANY_NUMBER)))
+                .andExpect(jsonPath("$.item_options.delivery_method", is(DeliveryMethod.POSTAL.getJsonName())))
+                .andExpect(jsonPath("$.item_options.delivery_timescale", is(DeliveryTimescale.STANDARD.getJsonName())))
+                .andExpect(jsonPath("$.postal_delivery", is(true)))
+                .andExpect(jsonPath("$.quantity", is(QUANTITY)));
+
+    }
+
+    @Test
+    @DisplayName("Fails to create certified copy item that fails validation")
+    void createCertifiedCopyItemFailsToCreateCertifiedCopyItem() throws Exception {
+        final CertifiedCopyItemRequestDTO certifiedCopyItemDTORequest = new CertifiedCopyItemRequestDTO();
+        final FilingHistoryDocumentRequestDTO filingHistoryDocumentRequestDTO = new FilingHistoryDocumentRequestDTO();
+
+        final CertifiedCopyItemOptionsRequestDTO certifiedCopyItemOptionsDTORequest = new CertifiedCopyItemOptionsRequestDTO();
```

Comment:
Could this class be formatted to match CH Java coding standards please? https://github.com/companieshouse/styleguides/blob/master/java.md